app/locales/en/translations.js

```diff
@@ -488,6 +488,11 @@ export default {
     'thisReportIsNoLongerAvailable': 'This report is no longer available.',
     'thursday': 'Thursday',
     'time': 'Time',
+    'timedRelease': 'Timed Release',
```

Comment:
this language string exists also in the common repo. your non-english translations might be better, so let's roll these into common afterwards.
this is a non-blocking issue, just making a note of it.